users/for_both_user_types/bothUserTypeModels.js

```diff
@@ -13,6 +14,12 @@ async function add(user) {
     return findById(id);
 }
 
+//function to find the user by id
+
+function findById(id) {
+    return db("users").where({ id }).first();
+}
+
 // function to get all users by type
 function findByUserType(type) {
     return DB("users").where({ user_type: type });
```

Comment:
Awesome, you fixed it!! Can we change the uppercase 'DB' to a lowercase 'db' on line 25 before merging? I had issues with the deployed version where the different versions of 'DB' and 'db' were breaking it.